docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -3610,28 +3610,33 @@ flag ElidableAxisValueName OlderSiblingFontAttribute;
 
 The `location` statement takes several formats:
 
+<a name="stat_loc_format_a"></a>
 ##### location format A (used in Axis value table [Format 1](https://docs.microsoft.com/en-us/typography/opentype/spec/stat#axis-value-table-format-1) and [Format 4](https://docs.microsoft.com/en-us/typography/opentype/spec/stat#axis-value-table-format-4))
 ```fea
 location <axisTag> <value>;+
 ```
 
 `axisTag` is a 4-letter tag and must correspond to one of the defined design
 axes in the table. `value` is a signed number specified as decimal (with
-optional fractional part) in the range -32767.0 to +32767.99998.
+optional fractional part) in the range -32768.0 to +32767.99998.
 
 With a single `location` statement the `AxisValue` will be format 1.
 If there are more than one `location` statements the `AxisValue` will be 
 format 4.
 
+<a name="stat_loc_format_b"></a>
 ##### location format B (used in Axis value table [Format 2](https://docs.microsoft.com/en-us/typography/opentype/spec/stat#axis-value-table-format-2))
 ```fea
 location <axisTag> <nominalValue> <rangeMinValue> - <rangeMaxValue>;
 ```
 
-Format for `axisTag` and the other values as above, and there must be a space
-before the `-` to distinguish it from a possible minus sign for
-`<rangeMaxValue>`. To specify an open ended range use `-32767` to mean 
-negative infinity and `32767.99998` to mean positive infinity. 
+Format for `axisTag` and `value`s is the same as for [location format
+A](#stat_loc_format_a) above. Note that the hyphen (`-`) defining the range must have spaces before
+and after so it does not get treated as a minus sign for `<rangeMaxValue>`.
```

Comment:
Space before is not required by the current grammar, someone reading this might make an implementation incompatible with makeotf.